lib/Multimap.js

```diff
@@ -14,19 +14,11 @@
  * limitations under the License.
  */
 
-/**
- * @template K, V
- */
 class Multimap {
   constructor() {
-    /** @type {!Map<K, !Set<!V>>} */
     this._map = new Map();
   }
 
-  /**
-   * @param {K} key
-   * @param {V} value
```

Comment:
@JoelEinbinder just curious... TSC didn't need these annotations? And how did you know they were unnecessary ?

Reply:
It’s a bug in typescript. https://github.com/Microsoft/TypeScript/issues/18135